frontstage/templates/register/register.confirm-organisation-survey.html

```diff
@@ -11,7 +11,7 @@ <h1 class="saturn">Confirm organisation and survey details</h1>
 
 <br/>
 
-<a href="{{ url_for('register_bp.register_enter_your_details', encrypted_enrolment_code=context.encrypted_enrolment_code) }}" class="btn action-button btn--primary">
+<a id="CONFIRM_BUTTON" href="{{ url_for('register_bp.register_enter_your_details', encrypted_enrolment_code=context.encrypted_enrolment_code) }}" class="btn action-button btn--primary">
```

Comment:
this should probably be lower case just to keep it consistent ( pedantic sorry :) )